packet/flatcar-linux/kubernetes/cl/controller-install.yaml.tmpl

```diff
@@ -5,10 +5,14 @@ systemd:
       enable: true
       contents: |
         [Unit]
+        StartLimitBurst=5
+        StartLimitIntervalSec=3600s
         Requires=network-online.target
         After=network-online.target
         [Service]
         Type=simple
+        Restart=always
+        RestartSec=60
```

Comment:
I think we want to have a comment explaining why these option were added. But explain why it is there, for example: we've seen that 1/3 of the times the download fails only on packet and just downloading it again a few times makes it work. Or whatever reason.

Just to make sure it is obvious why this line is there

Reply:
It's in the commit message.

Reply:
Oh, didn't get it is temporary. Why is it temporary? If we should remove it in some weeks/months, maybe we should create an issue to remind us (or a comment in the code, so someone sees it and removes it in the future? :)).

In any case, this is totally fine by me :)